framework/tlvf/yaml/tlvf/WSC/eWscDev.yaml

```diff
@@ -0,0 +1,13 @@
+#
+---
+_namespace: WSC
+
+eWscDev:
+  _type: enum
+  _enum_storage: uint16_t
+  WSC_DEV_NETWORK_INFRA_AP: 1
+  WSC_DEV_NETWORK_INFRA_ROUTER: 2
+  WSC_DEV_NETWORK_INFRA_SWITCH: 3
+  WSC_DEV_NETWORK_INFRA_GATEWAY: 4
+  WSC_DEV_NETWORK_INFRA_BRIDGE: 5
+  WSC_DEV_NETWORK_INFRA: 6
```

Comment:
That NETWORK_INFRA is 6 is just a coincidence: NETWORK_INFRA is a category enum, while NETWORK_INFRA_* are subcategory enums. So these should be defined in two separate enums.